packages/repository/src/model.ts

```diff
@@ -156,6 +156,16 @@ export class ModelDefinition {
     );
     return idProps;
   }
+
+  /**
+   * Returns the primary key(s) of a given model.
+   * Implemented by Raphael Drai for the purpose of PUT operations
```

Comment:
Probably best to leave names out of the source. I've seen others leave usernames with `TODO` comments

Reply:
HI, I am removing the PUT operation that will be handled in a separated PR.

Reply:
Please revert this block, we don't want to add a method that's commented out.

If you want to preserve this code for your future work, then you can create a new `git` branch:

```
# Create a new branch
$ git checkout -b hasOneCRUD-backup
$ git checkout hasOneCRUD
# Remove the commented out code now
$ git commit
```